src/status_im/ethereum/transactions/core.cljs

```diff
@@ -132,8 +132,7 @@
   [{:keys [db] :as cofx} {:keys [hash id address] :as transfer}]
   (let [transfer-by-hash (get-in db [:wallet :accounts address :transactions hash])
         transfer-by-id   (get-in db [:wallet :accounts address :transactions id])]
-    (when-let [unique-id (when-not (or transfer-by-id
-                                       (= transfer transfer-by-hash))
+    (when-let [unique-id (when-not (= transfer transfer-by-hash)
```

Comment:
are you sure about removing this condition?

Reply:
I'm not even sure what it was for, but then it works

Reply:
You mean it didn't work with it? Cause I think it might be for the case where you have multiple transfers in one single transaction?

Reply:
@yenda yep, pending transaction wouldn't be updated, but now it works